newrelic-agent/src/main/java/com/newrelic/agent/attributes/AgentAttributeSender.java

```diff
@@ -34,6 +34,10 @@ protected Map<String, Object> getAttributeMap() {
     }
 
     public void removeAttribute(String key) {
+        Map<String, Object> attributeMap = getAttributeMap();
```

Comment:
you can use the attributeMap from this line in line 40. Will save 1 method call until JIT optimizes.

Reply:
Yup, totally forgot to change that.